### PR TITLE
y_lb-update-donate-carousel-blocks|Updated styling of the general com…

### PR DESCRIPTION
**Pull request by oleh-rymar**
201
Updated styling of carousel and donate blocks.

### CODE REVIEW

File: assets/scss/carousel.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+@import "_functions";
+@import "_mixins";
+@import "_variables";
+
+.block-lb-carousel {
```
Comment (anpolimus) [inline assets/scss/carousel.scss:5]: @oleh-rymar you have to work with styles of this component in the module file for this component
https://git.drupalcode.org/project/lb_carousel/-/blob/2.0.x/assets/scss/lb-carousel.scss?ref_type=heads

File: assets/scss/donate.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+@import "_functions";
+@import "_mixins";
+@import "_variables";
+
+.block-lb-donate.block {
```
Comment (anpolimus) [inline assets/scss/donate.scss:5]: https://git.drupalcode.org/project/y_donate/-/blob/2.0.x/modules/lb_donate/assets/scss/lb-donate.scss?ref_type=heads

**Review by anpolimus — CHANGES_REQUESTED** [review]

File: assets/scss/carousel.scss (hunk 1 above)
Comment (oleh-rymar) [reply assets/scss/carousel.scss:5]: https://git.drupalcode.org/project/lb_carousel/-/merge_requests/29/diffs

File: assets/scss/donate.scss (hunk 2 above)
Comment (oleh-rymar) [reply assets/scss/donate.scss:5]: @anpolimus 
https://git.drupalcode.org/project/y_donate/-/merge_requests/14

File: assets/scss/carousel.scss (hunk 1 above)
Comment (froboy) [reply assets/scss/carousel.scss:5]: https://www.drupal.org/project/lb_carousel/issues/3438099#comment-15577663 tested and merged.

Comment (froboy) [reply assets/scss/carousel.scss:5]: (now with library version bump)

File: assets/scss/donate.scss (hunk 2 above)
Comment (froboy) [reply assets/scss/donate.scss:5]: https://www.drupal.org/project/y_donate/issues/3438104 merged. Same as above, please remember to bump the library version.

**Comment by froboy** [comment]
These are completed as per #201. This code is no longer needed.

**Pull request closed without merging**